cmd/template/framework/files/routes/fiber.go.tmpl

```diff
@@ -1,18 +1,23 @@
 package server
 
 import (
+	"time"
   {{if .AdvancedOptions.websocket}}
 	"context"
 	"log"
 	"fmt"
-	"time"
   {{end}}
 	"github.com/gofiber/fiber/v2"
+	"github.com/gofiber/fiber/v2/middleware/limiter"
   {{.AdvancedTemplates.TemplateImports}}
 )
 
 func (s *FiberServer) RegisterFiberRoutes() {
+	// Note: This can now be customized using parameters, making it easier to maintain and reusable.
+	rateLimiter := NewRateLimiter(5, 1 * time.Second)
 	s.App.Get("/", s.HelloWorldHandler)
+	// Note: The rateLimiter requires a second parameter because it won't work if placed after the handler.
```

Comment:
I'm not sure I understand this comment. Can you link to the documentation explaining this?